### ISSUE TITLE
HTML I&O replacement disables URL tooltip add-ons

### ISSUE THREAD

**Issue by Rekur** [Woundorf/foxreplace#264]
Firefox 64.0.2 x64 / Win7/SP1/x64 / FoxReplace 2.2.2

First of all, thank you for an excellent extension! Now...

Issue: any replacement performed in HTML mode "Input & output" essentially disables URL tooltips from extensions that provide such Link popups. This affects: [URL Tooltip WE](https://addons.mozilla.org/en-US/firefox/addon/url-tooltip-we/), [Link Status Redux](https://addons.mozilla.org/en-US/firefox/addon/link-status-redux/), [Link Status WE](https://addons.mozilla.org/en-US/firefox/addon/link-status-we/).

The issue can be observed on even the simplest sites, such as www.example.com: once any replacement (say, word "domain" replaced by "zyxabc") is performed in HTML I&O mode (only that mode), tooltips will no longer be shown when hovering over a link on that specific page - until it is refreshed or the relevant tooltip add-on is restarted.

Thank you once again!

**Comment by Woundorf** [comment]
Indeed, I have tried and those extensions fail when you replace with HTML I&O. I'll take a look on it at some later time, but compatibility with other specific extensions is not a priority.

**Comment by Rekur** [comment]
Thank you for your ultra-prompt response! Yep, I understand, and it's not a problem. I just figured I'd let you know in case you ever notice something that might be causing this.
